calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp

```diff
@@ -149,8 +164,9 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
 
   struct Parameters
   {
-    std::string radar_parallel_frame;  // frame that is assumed to be parallel to the radar (needed
-                                       // for radars that do not provide elevation)
+    std::string radar_optimization_frame;  // frame that is assumed to be parallel to the radar
+                                           // if estimating the transformation by 2d algorithms
+                                           // (needed for radars that do not provide elevation)
```

Comment:
I think that this is not true anymore and as such the description should change.

For example, base_link will be used as the parallel frame when calibrating the ARS548 right?

Reply:
Thanks, I rewrote the description in https://github.com/tier4/CalibrationTools/pull/180/commits/b2e2da78c451a706c76f9d02ceabf4d3b1b019fb